src/modules/tradeFormValidation/hooks/useGetTradeFormValidation.ts

```diff
@@ -0,0 +1,15 @@
+import { useAtomValue } from 'jotai'
+
+import useDebounce from 'legacy/hooks/useDebounce'
+
+import { tradeFormValidationAtom } from '../state/tradeFormValidationAtom'
+import { TradeFormValidation } from '../types'
+
+const TRADE_FORM_VALIDATION_DEBOUNCE = 200
+
+export function useGetTradeFormValidation(): TradeFormValidation | null {
+  const validation = useAtomValue(tradeFormValidationAtom)
+
+  // Just in case, to avoid blinking of the trade form button
+  return useDebounce(validation, TRADE_FORM_VALIDATION_DEBOUNCE)
```

Comment:
not sure if the debounce is in the right place. Not opposed to leaving this for now, and changing later given the timeline we have, but conceptually strange

Reply:
Sometimes trade form context changes very fast and we can see blinkings of the primary button.
To avoid this I added this debounce. Since it's a place here we get a value from an atom, it's a closest point where we can add a debounce.